index.html

```diff
@@ -1343,115 +1343,92 @@
-information about people, web servers, and other things. We can divide that
-information into three categories:
-
-1. Information that's fine to expose, for example because a person or group with
-    sufficient authority intended to expose that information or to do something that
-    necessarily exposes the information, or because it's not about people at all. For
-    example:
-
-    * The geolocation and camera APIs ask whether a person wants to expose their data.
-    * The URL a person is visiting must be sent to a server in order to navigate to that
-       URL, and known private-information-retrieval methods are too expensive to avoid that
-       exposure.
-    * The distribution of [[[largest-contentful-paint]]] timings for a website is about a website
-       rather than about the people browsing it, even if the data that informs that
-       measure can also reveal information about people.
-
-1. Information that we don't want to expose and have a plausible plan for removing access
-    to. For example, browsers are gradually removing the ability to join identities
-    between different [=partitions=].
-
-1. Information that we'd rather not expose, but that we don't have a plausible plan for
-    removing access to. For example:
-
-    * Some users are disappointed that the page they're visiting can discover which link
-        they clicked to leave that page. We can't block that information
-        because the page can use <a data-cite="RFC9110#status.3xx">HTTP redirects</a> to
-        learn it, and redirection is a core feature of the web.
-    * Some users are disappointed that a page with permission to run JavaScript can record
-        their pattern of interaction with that page. However, the page does this by using
-        the same events it would use to make the page interactive, so we can't block this
-        information access either.
-
-Some functionality of the web has historically been provided using functional primitives (e.g.
-third-party cookies) that can undermine people's privacy. As explained in <em>Improving the web
-without third-party cookies</em> ([[?web-without-3p-cookies]]), "<em>It is better to approach [these
-use cases] with replacement technologies that are designed-for-purpose and built to respect user
-privacy.</em>"
-
-The following subsections discuss how to review an API proposal that exposes data that
-provides a new way to infer each of the above categories of information. They explain how
-to <a data-cite="design-principles#leave-the-web-better">leave the web better than you
-found it</a>.
-
-### Handling acceptable information {#acceptable-information}
-
 <div class="practice">
-  <span class="practicelab">New APIs can add ways of getting acceptable
-  information that are guarded at least as strongly as the existing ways.</span>
+  <span class="practicelab">Web APIs must not weaken [=access guards=] which
+  protect users from [=inappropriate=] exposure of their information.</span>
```

Comment:
Oh, right, I wanted to think about the principle itself after I read the whole new section. I originally wrote 3 principles because we'd figured out that there were 3 kinds of new APIs that got proposed, and each one implied a different set of constraints. You've covered the constraints in the section, and I'm not super concerned about whether we describe the constraints as principles or just in prose. The remaining question is how to summarize the point of the overall section, and I'm not sure this is actually the right principle for that...

Maybe it's more like:

```suggestion
  <span class="practicelab">New Web APIs must guard users' information at least
  as well as existing APIs that are expected to stay in the web platform.</span>
```

Or something about the "target state of the web platform", but "target state" might need too much explaining.

Reply:
Or "at least as well as our concrete goals for the web platform"?

Reply:
I think the suggested edit is clear. I wouldn't want to complicate it by referring to "target state" or "concrete goals". This might want more noodling on by the group though, so I won't add the commit right now.